### PR TITLE
Complete term entry for ACOS SQL function

### PR DESCRIPTION
**Pull request by jubalclary**
### Description

I have wrote a term entry for the ACOS function in SQL for the issue "[Term Entry] SQL Math Functions ACOS #2265".

The file path to my changes is docs/content/sql/concepts/math-functions/terms/acos/acos.md

I *heavily* relied on the doc page for the COS function: https://github.com/Codecademy/docs/blob/main/content/sql/concepts/math-functions/terms/cos/cos.md

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/sql/concepts/math-functions/terms/acos/acos.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+CatalogContent:
+    - 'learn-sql'
+    - 'paths/analyze-data-with-sql'
+---
+
+The SQL function **`ACOS()`** calculates the arccosine of a given value, expressed in radians.
+
+## Syntax
+
+```sql
```
Comment (dakshdeepHERE) [inline content/sql/concepts/math-functions/terms/acos/acos.md:22]:
```suggestion
```pseudo 
```
With syntax always use pseudo.

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
A change from my side.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@jubalclary I did a first review and left a few comments.

**Comment by jubalclary** [comment]
Thanks for the feedback, I just committed some changes

**Review by SSwiniarski — APPROVED** [review]
@jubalclary I made a small typo change, now it looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@jubalclary @SSwiniarski Looks good to merge. I just made a couple of minor edits.

**Comment by SSwiniarski** [comment]
@jubalclary Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/sql/math-functions/acos

image

You're currently anonymous. If you don't wish to be anonymous, check out this forum thread on how to link your GitHub and Codecademy profiles: https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036  **Note:** it may take a few minutes for the change to register on the entry page.

**Pull request merged**